tests/generate_test_data.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os
+import json
+from mwdblib import Malwarecage  # type: ignore
+from src.api import parse
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Generation of json and txt test files for parsing test"
+    )
+    parser.add_argument("mwdb_user", help="Mwdb username")
+    parser.add_argument("mwdb_pass", help="Mwdb password")
+    parser.add_argument("mwdb_config_id", nargs="?",
```

Comment:
I don't mind the formatting (it's OK), but for the sake of consistency, do you mind `black`ing this file?